Project 3 Lol

```diff
@@ -0,0 +1,350 @@
+  - Unless its founders or relevant organizers dissolve it after weeks of nobody attending, Anime Club continues to exist as an organization with no members.
+  
+  (e) Any site is partially bounded by some instance of Material Entity.
+  - True: a fully immaterial site, one not bounded to anything 'out there', cannot exist in the world. 
+  - I am under the impression that a site is a 'de jure' entity; it only exists because we pay special attention to it.
+  - We cannot demarcate a site around something wholly immaterial. 
+  
+  (f) A book placed under the leg of a wobbly table has acquired a new function. 
+  - False: function is a specifically dependent quality; it is unique to the continuant which bears it.
+  - Many other objects can be placed under the table, like a brick, or two other, thinner books.
+  - Because stabilizing a table is a generic quality, the book has acquired a new role.
+  
+  (g) A glass vase cushioned with packing tape for all time, has the disposition to break. 
+  - True: the glass vase itself has the disposition to break, regardless of individual circumstance.
+  - The 'vase-and-packing-tape' could be an instance of a new type of 'cushioned object', which may not have the disposition to break.
+  
+  (h) Spacetime is a class in BFO.
+  - False: I cannot provide an explanation using theories in physics.
+  - I have an intuition that spacetime cannot be represented in BFO because a spacial region is a continuant while a temporal region is an occurrent.
+...
+  - Looking at BFO in Protege I can now see that "spatiotemporal region" is an occurrent class in bfo made for regions of spacetime.
+  - Spacetime itself cannot be an instance of a region (unsure).
+  
+  (i) The continuant fiat boundary class of BFO is closed, meaning, there are no subclasses beyond those identified presently in BFO.
+  - I support Finn's assertion in the project 3 google doc that any of the given fiat boundary subclasses can be further subclassified according to a given domain and according to different granularities.
+  - Like, measurements of a fiat boundary could subclassify them further, for example.
+  
+```
+8. Model the following scenario in BFO, introducing whatever terms are needed to do so: John runs for 3 hours, startin slowly, speeding up during the middle, then ending the run at a slower pace.  
+- John instance_of object
+- John_run instance_of process
+- John participates_in John_run
+- John_runtime instance_of generically dependent continuant
+...
+- John_runpace instance_of process profile
+- John_runpace_warmup part_of John_runpace 
+- John_runpace_stride part_of John_runpace 
+- John_runpace_cooldown part_of John_runpace 
+...
+- John_runnerhigh is_a quality
+- John has_characteristic John_runnerhigh
+^^ How would I add temporal parameters to this, clarifying that John only has a runners high after John_run is completed?
```

Comment:
You probably need to order processes one after the other, and then add something like a zero-dimensional temporal region or one-dimensional temporal region and say that the property is had by John only there. Take a look at what we did on the google doc